maths/distance_formula.py

```diff
@@ -0,0 +1,15 @@
+def distance_formula(x1, y1, x2, y2):
```

Comment:
add type hints.
Change function name and file name to "euclidean_distance" or whatever distance formula this is. Update the comments to reflect this change.